src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -262,8 +262,18 @@ BEGIN
   IDS_DEFAULTPWLENGTHTOOSMALL "Password length is less than sum of 'at least' constraints"
   IDS_CANTSPECIFYMINNUMBER "Sorry, can't specify a minimum number of characters with 'pronounceable' option. Specified values ignored."
   IDS_OWNSYMBOLSMISSING    "If you specify using your own symbol list, you cannot leave it empty. Unselecting use of symbols."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_BEFORE "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
-  IDS_EXCLUDE_FROM_SCR_CAP_OVERRIDE_WARNING_AFTER  "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nClick OK to continue."
+
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_INVALID "Screen capture protection disabled by unknown cause (state error).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_NONE "Screen capture protection is enabled."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_CMDLINE "Screen capture protection disabled by command line.\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_ICA "Screen capture protection disabled by detection of the ICA protocol (i.e., Citrix).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_RDP "Screen capture protection disabled by detection of the RDP protocol (i.e., Windows Remote Desktop).\n\nContact support if this is unexpected.\n\nYou can click OK to continue in %d seconds."
+
+  IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_AFTER_INVALID "Screen capture protection disabled by unknown cause (state error).\n\nContact support if this is unexpected.\n\nClick OK to continue."
```

Comment:
See comment for IDS_SCRCAP_EXCLUDE_OVERRIDE_WARNING_BEFORE_INVALID

Reply:
_INVALID is displayed when a reason is not known, caused by a program bug, system corruption, or other such rare event we should never see. It is an error handling value that is present for consistency in handling known good values while having something for the error case (instead of a hard-coded value in source). See areas where this resource is used. All of these have the same pattern: invalid_value,known_good1,known_good2, etc.